libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -455,7 +470,44 @@ class BreezSDKMapper {
             "message": messageSuccessActionData.message
         ]
     }
-    
+
+    static func dictionaryOf(receivePaymentRequest: ReceivePaymentRequest) -> [String: Any] {
+        return [
+            "amountSats": receivePaymentRequest.amountSats,
+            "description": receivePaymentRequest.description,
+            "preimage": arrayOf(preimageBytes: receivePaymentRequest.preimage),
+            "openingFeeParams": dictionaryOf(params: receivePaymentRequest.openingFeeParams)
+        ]
+    }
```

Comment:
This is not needed, but an asReceivePaymentRequest function should be added to convert a dictionary to ReceivePaymentRequest.

Reply:
Done as part of a1b1947